### ISSUE TITLE
Dialog bod and options for 'copy framework' frame

### ISSUE THREAD

**Issue by brandonopened** [opensalt/opensalt#287]
1) Add button for 'copy selected' https://opensalt.mybalsamiq.com/projects/opensalt/Tree%20view_%20copy
(on CF item tree view page) 
2) have drag+drop a framework or the new button bring up a dialog box much like "Make Associations" does and for now, allow simply option to either create derivative framework (will create NEW identifiers with exact match of relationships and retain all current IsChildOf associations)
    https://opensalt.mybalsamiq.com/projects/opensalt/copy_dialogbox

**Comment by brandonopened** [comment]
1) copy identifier and keep ExactMatch/other associations 
2) copy but create new identifier; no exact match ? @jmarks

**Comment by brandonopened** [comment]
Existing copy framework behavior: Creates a new identiifer. So for "derivative" we are expanding that to also create the Exact Match Of relationship. 

For the "Simple Copy", we are just displaying the framework on our server from the external source. I do not know how to describe this technically, could use some help here @roverwolf @jmarks

**Comment by jmarks** [comment]
Make a simple copy is making a duplicate without any exactMachOf relationships, while making a derivative copy includes the exactMatchOf relationships. In both cases a new item is derived from the thing it is copied from.

**Comment by roverwolf** [comment]
Trying to summarize to make sure I have it correct.

We have:
1. Copy content (new identifiers, **no** associations copied)
2. Copy and associate with original (new identifiers, exactMatchOf created to copied item)

Do we need to also deal with:
- Copying all children of the item (all vs *n* levels?)
- Copying only the selected children (keeping parent info) since the UI allows children to be selected
- Copying and creating **all** (in framework, non isChildOf) associations of original (plus new exactMatchOf -- or not?)

**Comment by brandonopened** [comment]
I'm seeing this ticket as the MVP for copying from an external server. Get the dialog box made... yes, we already do what you call option 1 (at least when copying from the same server). So with this ticket we are changing the security options to allow that operation to happen and adding option two. 

I envision a checkbox/more extensive logic we can add later to address the other three or more scenarios.

**Comment by roverwolf** [comment]
Okay, was trying to clairify the scope since the issue is titled with dialog box and options.

What we are talking about here is basically #127 then?

**Comment by brandonopened** [comment]
Yes, we came back to the same conclusion. I linked the balsamiq mockup here as well though.

**Comment by brandonopened** [comment]
Ok, will split into fourthings:
1) add license uri and license-text on create cf doc
1b) add fields to cfdoc page for editing
2) have it display on the cf tree page (?)
3) make sure it exports correctly When added

**Comment by brandonopened** [comment]
https://www.youtube.com/watch?v=57QIp8M0HI4
